scripts/generate-changed-recipes

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/python
```

Comment:
```suggestion
#!/usr/bin/env python
```

And maybe add `.py` extension to file so you know at glance it's Python script?

Reply:
0afb3a1e76cb0fcb892a62b148ac00ced4a3524f